source/braille.py

```diff
@@ -2713,6 +2713,12 @@ def _get_keyNames(self):
 		@rtype: list
 		"""
 		return self.id.split("+")
+	
+	def _get_speechEffectWhenExecuted(self):
+		if not config.conf["braille"]["interruptSpeechWhileScrolling"]:
+			if self.script and self.script.__name__.startswith("script_braille_scroll"):

```

Comment:
In the case that add-ons use that prefix, unexpected behaviour may occur, confusing the developer or users.
I think it's worth considering adding a generic property for scripts (eg tags) and using that to determine if this is a braille scroll gesture. This allows addons that introduce braille scroll scripts to opt into this feature.

Reply:
Otherwise, it's probably fine to note that scripts starting with `script_braille_scroll` work with `interruptSpeechWhileScrolling` in the developerGuide somewhere.

Reply:
Fixed in 2e894fe